### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by jbpiacentino**
## Describe your PR

First pass at reviewing the KC addon doc, mostly from a marketing stand-point
- Broken english may remain
- Not sure about Console action description terminology - need proofing
- 

## Checklist

- [ ] My PR is related to an opened issue : alliances/marketplace/product#1
- [ ] I've read the [contributing guidelines](../CONTRIBUTING.md)

## Reviewes
_Who should review these changes?_ @juliamrch @vballu

### CODE REVIEW

File: content/doc/addons/keycloak.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+type: docs
+title: Keycloak
+shortdesc: This add-on provides a one-click Keycloak single sign-on and identity and access management (IAM) solution based on existing Clever Cloud services and taking advantage of the *Please open-it* experiences, our Keycloak expert partner.
```
Comment (vballu) [inline content/doc/addons/keycloak.md:4]: `shortdesc`-> `description`

Comment (juliamrch) [reply content/doc/addons/keycloak.md:4]:
```suggestion
description: This add-on provides a one-click Keycloak single sign-on and identity and access management (IAM) solution based on existing Clever Cloud services and taking advantage of the *Please open-it* experiences, our Keycloak expert partner.
```

File: content/doc/addons/keycloak.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+### Realms management
+We recommend using __as few Realm as possible__ : they may significantly impact the overall performances.
+
+Realms can be managed in two way
+1. __RECOMMENDED:__ using Clever Cloud's Keycloak add-on environment variables
+- In your Keycloak java application, go to the `Environment variables` panel and set as many Realms you need, separate by `,`.
+- Rebuild and redeploy your Java application after the change
+
+> to deploy two realm `realm_1`, `realm_2` just set `REALMS=realm_1,realm_2`
```
[the remark this answers is not in the document]
Comment (juliamrch) [reply content/doc/addons/keycloak.md:109]:
```suggestion
> to deploy two realms `realm_1`, `realm_2` just set `REALMS=realm_1,realm_2`
```

File: content/doc/addons/keycloak.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+Keycloak on Clever Cloud is the easiest way to set up a ready-to-use Keycloak.
+
+It starts with
+- Java 17: XS
+- PostgreSQL: XS Tiny Space
+- FSBucket
+
+We add services management fees for maintenance, configuration, platform costs, and support. During the technical review, this service management is free.
+
+Hence, the total price by month for the base plan is (according to the [pricing page](https://www.clever-cloud.com/pricing/)):
```
[the remark this answers is not in the document]
Comment (juliamrch) [reply content/doc/addons/keycloak.md:151]:
```suggestion
Hence, the total price for 30 days for the base plan is (according to the [pricing page](https://www.clever-cloud.com/pricing/)):
```

File: content/doc/addons/keycloak.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+- `JAVA XS` (16€) + `PostgreSQL XS Tiny Space` (15€) + `< 100MB FSBucket` (0€) + `add-on management fees` (O€) = 31€/month
```
[the remark this answers is not in the document]
Comment (juliamrch) [reply content/doc/addons/keycloak.md:152]:
```suggestion
- `JAVA XS` (16€) + `PostgreSQL XS Tiny Space` (15€) + `< 100MB FSBucket` (0€) + `add-on management fees` (O€) = 31€/30 days
```

**Pull request closed without merging**